src/server/room-controller.js

```diff
@@ -86,6 +73,34 @@ export default class RoomController {
     return res.status(201).end();
   }
 
+  async handleGetRoomPage(req, res) {
+    const { roomId } = req.params;
+    const room = await this.roomService.getRoomById(roomId);
+
+    if (!room) {
+      throw new NotFound();
+    }
+
+    const { _id: userId } = req.user;
+
+    let invitations = null;
+
+    if (room.access === ROOM_ACCESS_LEVEL.private) {
+      const isOwnerOrMember = await this.roomService.isRoomMemberOrOwner(roomId, userId);
```

Comment:
Tiny thing, but I find it unnecessary to reverse the orders or "owner" and "member" (`isOwnerOrMember` vs `isRoomMemberOrOwner`).
Could we please streamline? I'd vote for renaming `isRoomOwnerOrMember` ✋

Reply:
Fixed and renamed everything to isRoomOwnerOrMember, it sounds better indeed.

Reply:
Yeah it makes more sense, from order of interest/priority, Owner before Member.
Cool, thanks!